Content.Server/Construction/ConstructionSystem.Initial.cs

```diff
@@ -203,16 +207,31 @@ void ShutdownContainers()
                             if (used.Contains(entity))
                                 continue;
 
+                            // Dump out any stored entities in used entity
+                            if (TryComp<SharedStorageComponent>(entity, out var storage) && storage.StoredEntities != null)
+                            {
+                                foreach (var storedEntity in storage.StoredEntities.ToList())
+                                {
+                                    var transform = Transform(storedEntity);
```

Comment:
Could you help me double check that replacing lines 215-219 with a call to `Content.Server/Storage/EntitySystems/StorageSystem.cs:RemoveAndDrop` does the same thing? It looks like it correctly handles some more book-keeping logic related to storage systems.

Reply:
Good find. This has been integrated